### PR TITLE
Feature/update access SMILE operators 

### PR DESCRIPTION
**Pull request by nikhil**
- This is a pull request to mirror the changes pushed to develop here (#1006)

### CODE REVIEW

File: runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+from runner.run.objects.run_creator_object import RunCreator
+from file_system.repository.file_repository import FileRepository
+
+
+logger = logging.getLogger(__name__)
+WORKDIR = os.path.dirname(os.path.abspath(__file__))
+
+METADATA_OUTPUT_FIELDS = [
+    "barcodeId",
+    "sampleName",
```
Comment (sivkovic) [inline runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py:18]: Should we reference some of the metadata keys from `settings.py`

File: runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    for sample_group in samples_groups:
+        meta = sample_group[0]["metadata"]
+        meta.update(
+            {
+                "dnaInputNg": calc_avg(sample_group, "dnaInputNg"),
+                "captureInputNg": calc_avg(sample_group, "captureInputNg"),
+                "libraryVolume": calc_avg(sample_group, "libraryVolume"),
+                "libraryConcentrationNgul": calc_avg(sample_group, "libraryConcentrationNgul"),
+                "captureConcentrationNm": calc_avg(sample_group, "captureConcentrationNm"),
+                "requestId": request_id,
```
Comment (sivkovic) [inline runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py:77]: same here?

File: runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py (hunk 1 above)
Comment (nikhil) [reply runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py:18]: Added a commit to address this

File: runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py (hunk 2 above)
Comment (nikhil) [reply runner/tests/operator/cmo_ch/v2_1_0/nucleo/nucleo_operator.py:77]: Added a commit to address this

**Review by sivkovic — APPROVED** [review]

**Pull request merged**